### ISSUE TITLE
Building for web fails due to nullable locale.

### ISSUE THREAD

**Issue by Joran-Dob** [wiredashio/wiredash-sdk#129]
Thanks for building this great product, we use it now in a bunch of apps we published and it has been great! 🎊 

 Building for web fails due to nullable locale. When trying to build the following errors are thrown. 

```dart
../../../../.pub-cache/hosted/pub.dartlang.org/wiredash-0.6.0/lib/src/common/options/wiredash_options_data.dart:16:33: Error: A value of type 'Locale?' can't be assigned to a variable of type 'Locale' because 'Locale?' is nullable and 'Locale' isn't.
 - 'Locale' is from 'dart:ui'.
        _currentLocale = locale ?? window.locale,
                                ^

../../../../.pub-cache/hosted/pub.dartlang.org/wiredash-0.6.0/lib/src/common/options/wiredash_options_data.dart:63:31: Error: A value of type 'Locale?' can't be assigned to a variable of type 'Locale' because 'Locale?' is nullable and 'Locale' isn't.
 - 'Locale' is from 'dart:ui'.
      _currentLocale = window.locale;

```

wiredash version: 0.6.0
flutter version: 2.0.0

**Comment by Joran-Dob** [comment]
Apparently, this causes the same build issue for both Android and iOS. Not sure if it is only happening when unsound nullsafety flag is not set.

**Comment by Joran-Dob** [comment]
Fixed in dev branch by merged pull #130
